### PR TITLE
hack: add license checker and add license to files that failed the checker

### PR DESCRIPTION
**Pull request by fanminshi**

### CODE REVIEW

**Comment by fanminshi** [comment]
cc/ @hongchaodeng @hasbro17

File: hack/test [hunk 1]
```diff
@@ -66,6 +66,19 @@ function fmt_pass {
 	else
 		echo "Skipping unused: failed to install"
 	fi
+
+	echo "Checking for license header..."
+	licRes=""
+	files=$(find . -type f -iname '*.go' ! -path './vendor/*')
```
Comment (hongchaodeng) [inline hack/test]: use `$allfiles`

**Comment by fanminshi** [comment]
all fixed. PTAL cc/ @hongchaodeng

**Comment by fanminshi** [comment]
fixed again.

**Comment by hongchaodeng** [comment]
lgtm

**Comment by fanminshi** [comment]
merge when green.

**Pull request merged**